### ISSUE TITLE
Does it support the ARM architecture?

### ISSUE THREAD

**Issue by 51Hertz** [ossrs/srs#4113]
Do the experts know if CentOS can be deployed on an ARM architecture?

`TRANS_BY_GPT4`

**Comment by suzp1984** [comment]
yes, check the `OS/ARCH` of CentOS docker image: https://hub.docker.com/_/centos/tags

**Comment by 51Hertz** [comment]
I deployed KubeSphere on an ARM architecture and then attempted to deploy Oryx, but encountered an error.

`TRANS_BY_GPT4`
